nginx_proxy/config.yaml

```diff
@@ -26,6 +26,7 @@ options:
     active: false
     default: nginx_proxy_default*.conf
     servers: nginx_proxy/*.conf
+  real_ip_from: []
```

Comment:
<https://github.com/home-assistant/addons/blob/master/nginx_proxy/DOCS.md#configuration>
The configuration example has no value and should probably be adjusted as well.

Reply:
Good point, added that!